easybuild/tools/py2vs3/__init__.py

```diff
@@ -34,26 +32,3 @@
 
 _log = fancylogger.getLogger('py2vs3', fname=False)
 _log.deprecated("Using py2vs3 is deprecated, since EasyBuild no longer runs on Python 2.", '6.0')
-
-
-def python2_is_deprecated():
```

Comment:
The functionality provided by `easybuild.tools.py2vs3` is deprecated, but why would we hard break the API just to clean up some code at this point?

`from easybuild.tools.py2vs3 import python2_is_deprecated` should keep working, but produce a deprecation warning

So I'm not in favor of this (unless I'm missing something that makes this cause trouble)

Reply:
380ea38